blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AbstractMetricsRW.java

```diff
@@ -74,6 +91,10 @@ protected synchronized void setLocatorCurrent(Locator loc) {
         insertedLocators.put(loc.toString(), Boolean.TRUE);
     }
 
+    protected synchronized void setDelayedLocatorForASlotCurrent(int slot, Locator locator) {
```

Comment:
The existing method name to cache locators is `setLocatorCurrent`. I named this method similar to that.